src/lift.js

```diff
@@ -4,7 +4,7 @@ var liftN = require('./liftN');
 
 /**
  * "lifts" a function of arity > 1 so that it may "map over" an Array or other
- * Functor.
+ * object that satisfies the [FantasyLand Apply spec](https://github.com/fantasyland/fantasy-land#apply).
```

Comment:
I can live with this.  If I ever get around to adding a Types section, I'll think about adding it there.